frontend/model/contracts/group.js

```diff
@@ -65,6 +70,12 @@ function initGroupProfile (contractID: string) {
   }
 }
 
+export function removeMemberSecret () {
+  return {
+    secret: `${parseInt(Math.random() * 10000)}` // TODO: this
+  }
+}
+
```

Comment:
1. Let's keep group.js uncluttered. It should ideally be only what's inside `DefineContract`. We already have too much stuff defined outside of it.
2. This function is only called once, and it doesn't do anything significant. Therefore it shouldn't be defined as a function, and instead its call to `Math.random().toFixed(0)` inlined.